api/ocm/tools/transfer/transferhandler/spiff/README.md

```diff
@@ -15,14 +14,14 @@ This mode is used to decide on the update option for a component version. This c
 
 - `mode` *&lt;string>*: `update`
 - `values` *&lt;map>*:
-  - `component` *&lt;map>*:  the meta dats of the component version carrying the resource
+  - `component` *&lt;map>*:  the meta data of the component version carrying the resource
     - `name` *&lt;string>*: component name
     - `version` *&lt;string>*: component version
-    - `provider` *&lt;provider>*: provider info, a struct with the field s
+    - `provider` *&lt;provider>*: provider info, a structure with the fields
```

Comment:
```suggestion
    - `provider` *&lt;provider>*: provider info, a struct with the fields
```

Theoretically, `struct` should be the correct term, since it is a `go struct`, right?

If not, please ignore the following suggestions^^